app/views/sephora_style_guide/home/index.slim

```diff
@@ -15,7 +15,9 @@ main.container-fluid#app role='main'
   = render 'sephora_style_guide/home/tabs'
   = render 'sephora_style_guide/home/accordion'
   = render 'sephora_style_guide/home/loaders'
-
+  = render 'sephora_style_guide/home/alerts'
+  = render 'sephora_style_guide/home/checkbox_radio'
```

Comment:
Checkbox / Radio buttons are a part of forms, so just to need to append them there.